src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.PowerShell.Commands
     /// The ConvertTo-Json command.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Validate Json serialization.Validate Json Serialization for 'Get-CimClass' and 'Get-Command'.Error occurred in Context block
```powershell
Cannot process argument transformation on parameter 'Pending'. Cannot convert value "System.String" to type "System.Management.Automation.SwitchParameter". Boolean parameters accept only Boolean values and numbers, such as $True, $False, 1 or 0.
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Json.Tests.ps1:1316 char:83
```